### ISSUE TITLE
java.lang.String cannot be cast to com.facebook.react.uimanager.AccessibilityDelegateUtil$AccessibilityRole - on 0.57.7

### ISSUE THREAD

**Issue by Arthuron** [facebook/react-native#22484]
Hi There,

i know this issue was tracked here
https://github.com/facebook/react-native/issues/21754
but for the version 0.57.3

Currently iam working with the version 0.57.7 and its not to easy to change this at the current project phase.

anyone know a solution for my problem, its the same like the linked ticket.
